dirbot/pipelines.py

```diff
@@ -10,7 +12,7 @@ class FilterWordsPipeline(object):
 
     def process_item(self, item, spider):
         for word in self.words_to_filter:
-            if word in unicode(item['description']).lower():
+            if word in six.text_type(item['description']).lower():
```

Comment:
Why isn't just `item['description']).lower()` enough?
`unicode(item['description'])` was just wrong, by the way :)

I think it is not good to add six dependency to example project because users shouldn't write Python 2+3 compatible code in most cases.